### PR TITLE
Add list of repos

### PR DESCRIPTION
**Pull request by cnmeow**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -418,17 +421,127 @@
...
+                  <li class="infoRepo"> Dùng 4 phím lên-xuống-trái-phải để điều khiển con ma. Trong khi di chuyển, nó sẽ vẽ 😗 </li>
+                 
+               </ul>
+
+            </div>
+
+         </div>
+
+        
+
```
Comment (tuanle03) [inline index.html]:
```suggestion
```
Remove some empty line

File: index.html [hunk 2]
```diff
@@ -418,17 +421,127 @@
...
+      const cntRepos = data.public_repos;
+      const cntFollowers = data.followers;
       document.getElementById('avatar').src = avtUrl;
-      document.getElementById('cntRepos').innerHTML = publicRepos;
+      document.getElementById('cntRepos').innerHTML = cntRepos;
+      document.getElementById('followers').innerHTML = cntFollowers;
    })
 </script>
 </body>
 </html>
```
Comment (tuanle03) [inline index.html]: Refactor your code

**Review by tuanle03 — CHANGES_REQUESTED** [review]

**Comment by tuanle03** [comment]
Please deploy branch `main` not this branch

**Comment by cnmeow** [comment]
Done

**Pull request closed without merging**